src/wwwroot/template-editor/adapterService.js

```diff
@@ -38,8 +41,32 @@
 
     function init() {
       langKey = getPreferredLanguage();
+      decodedToken = getToken();
```

Comment:
If I am not wrong, you do not need to decode the token to use it. So, maybe you do not need to include _jwtHelper_.

I know that you need some more information from the token, but maybe we could prepare it using the WebApp, for example storing the accountname and userid in the local storage.

Reply:
Or passing it as parameter

Reply:
I guess the reason to use the token is to not allow pass this information in plain text mode by the network, maybe we can store the extra data needed in localstorage but I guess we need to think in the security concerns.

Reply:
The token itself is the secret, so, it is already plain text.

_UserId_ and _accountname_ are not secrets.

Reply:
What is your security concern?

Reply:
My concerns was if UserId and accountname can be stored in plain text, but if it's not a problem I can update that to avoid inject the angular-jwt module. Thanks andres

Reply:
Finally we will use a login json object in the webapp to wrapping jwtToken and include the new properties for mseditor